### ISSUE TITLE
Archive this gem

### ISSUE THREAD

**Issue by opoudjis** [metanorma/metanorma-un#279]
We have agreed to a 15-flavour limit in maintaining separate gems, and we have already gone over that limit months ago with Plateau. This gem and CSA have had the least activity of any of our flavours, and I am not seeing a realistic prospect of reengaging with UN, from past conversations; there has been no real activity on this gem since 2020. 

We have already discussed in the past that this will be the next gem to archive, and I request that we now do so.

**Comment by opoudjis** [comment]
I am now enacting this, in light of the request to generate a new flavour, metanorma-pdfa.

@Intelligent2013 That means that as with metanorma-gb, metanorma-mpfa and metanorma-m3aawg, I will no longer be maintaining this gem, and you should stop maintaining its associated XSLT as well.

@CAMOBAP This means that, as with metanorma-gb, metanorma-mpfa and metanorma-m3aawg, this gem and its samples should no longer be tested under metanorma-cli

**Comment by CAMOBAP** [comment]
@opoudjis There is no `gb`, `mpfa` and `m3aawg` in the metanorma-cli's matrix already and all downstream deliverables like docker, packed-mn

- https://github.com/metanorma/metanorma-cli/commit/7ee73e29ebdd8278fad2fe6fcba17fcfbeb47a9c - `m3aawg` removed
-  `gb`, `mpfa` never was in our test matrix

@opoudjis  Because this ticket was created in `metanorma-un` is there any changes in test matrix required for `UN`?

**Comment by opoudjis** [comment]
> @opoudjis Because this ticket was created in `metanorma-un` is there any changes in test matrix required for `UN`?

No. If we ever reactivate metanorma-un, the internal test matrix can kick in again.
